### PR TITLE
Feature/zk UI 99 search iam users

### PR DESCRIPTION
**Pull request by JBWatenbergScality**

### CODE REVIEW

File: src/react/account/AccountUserList.jsx [hunk 1]
```diff
@@ -126,23 +172,43 @@ const AccountUserList = () => {
...
-          }
-        }}
-      >
+      <Table columns={columns} data={iamUsers} defaultSortingKey={'userName'}>
         <TableHeader>
-          <Table.SearchWithQueryParams
-            displayedName={{
-              singular: 'user',
-              plural: 'users',
-            }}
```
Comment (ChengYanJin) [inline src/react/account/AccountUserList.jsx:170]: Maybe we should keep this in order to display `Total {iamUsers.length} user(s)`?

File: src/react/account/AccountUserList.jsx [hunk 2]
```diff
@@ -158,7 +224,20 @@ const AccountUserList = () => {
...
-        />
+        >
+          {Rows => (
+            <>
+              {listUsersFirstPageStatus === 'loading' ||
+              listUsersFirstPageStatus === 'idle'
+                ? 'Loading users...'
+                : ''}
+              {listUsersFirstPageStatus === 'error'
+                ? 'We failed to retrieve users, please retry later and if the error persist contact your support.'
```
Comment (ChengYanJin) [inline src/react/account/AccountUserList.jsx]:
```suggestion
                ? 'We failed to retrieve users, please retry later. If the error persists, please contact your support.'
```

**Review by ChengYanJin — COMMENTED** [review]
I am good with the behavior!

**Review by MonPote — APPROVED** [review]

**Comment by MonPote** [comment]
ping

**Pull request merged**